core/src/main/java/io/aiven/klaw/config/ConfigUtils.java

```diff
@@ -46,7 +48,15 @@ protected static List<String> getStaticResources(boolean coralEnabled) {
       staticResourcesHtmlArray.add("/assets/coral/**");
     }
 
-    return staticResourcesHtmlArray;
+    AntPathRequestMatcher[] antPathRequestMatchersArray =
+        new AntPathRequestMatcher[staticResourcesHtmlArray.size()];
+    int i = 0;
+    for (String s : staticResourcesHtmlArray) {
+      antPathRequestMatchersArray[i] = new AntPathRequestMatcher(s);
+      i++;
+    }
```

Comment:
```suggestion
    int i = 0;
    for (int i = 0; i < staticResourcesHtmlArray.length; i++) {
      antPathRequestMatchersArray[i] = new AntPathRequestMatcher(s);
    }
```
This would be shorter